README.md

```diff
@@ -132,15 +132,18 @@ Use this option to customize the topmost banner image with the Fidel UI. If you
 Fidel.bannerImage = Bitmap(...);
 ```
 
-### country
+### allowedCountries
 
-To set a default country for the SDK you should use:
+To set the countries that the users can select, use
 
 ```java
-Fidel.country = Fidel.Country.UNITED_KINGDOM;
+Fidel.allowedCountries = new Fidel.Country[]{Fidel.Country.UNITED_KINGDOM, Fidel.Country.JAPAN, Fidel.Country.CANADA};
 ```
 
-When you set a default country, the card linking screen will not show the country picker UI. The other options, for now, are: `.UNITED_STATES`, `.IRELAND`, `.SWEDEN`, `.JAPAN`, `.CANADA`.
+The possible options are: `.UNITED_KINGDOM`, `.UNITED_STATES`, `.IRELAND`, `.SWEDEN`, `.JAPAN`, `.CANADA`. You can set zero, one or multiple of these countries. 
```

Comment:
Don't forget to modify the documentation, please. It should not be possible to set zero countries.